packages/shared/src/utils/contract/contract.ts

```diff
@@ -32,13 +32,15 @@ export const isValidToSell = (contract_info: TContractInfo) =>
 
 export const hasContractEntered = (contract_info: TContractInfo) => !!contract_info.entry_spot;
 
-export const isAccumulatorContract = (contract_type: string) => /ACCU/i.test(contract_type);
+export const isAccumulatorContract = (contract_type = '') => /ACCU/i.test(contract_type);
 
-export const isMultiplierContract = (contract_type: string) => /MULT/i.test(contract_type);
+export const isMultiplierContract = (contract_type = '') => /MULT/i.test(contract_type);
 
-export const isVanillaContract = (contract_type: string) => /VANILLA/i.test(contract_type);
+export const isVanillaContract = (contract_type = '') => /VANILLA/i.test(contract_type);
 
-export const isCryptoContract = (underlying: string) => /^cry/.test(underlying);
+export const isOnlyUpsDownsContract = (contract_type = '') => /RUN/i.test(contract_type);
+
+export const isCryptoContract = (underlying = '') => /^cry/.test(underlying);
```

Comment:
There is a code smell here and on line 143. It suggests to replace `/^cry/.test(underlying);` with `underlying.startsWith('cry')`. Not sure if we should do it, please feel free to share your opinion

Reply:
@kate-deriv please feel free to replace it, it's identical. please remember to add the places where these (updated) functions are used to QA checklist for sanity check.

Reply:
Done👍🏻
Also added to QA-checklist